internal/lookoutingester/lookoutdb/insertion_test.go

```diff
@@ -17,31 +17,34 @@ import (
 
 	"github.com/G-Research/armada/internal/lookout/testutil"
 	"github.com/G-Research/armada/internal/lookoutingester/model"
-	"github.com/G-Research/armada/pkg/armadaevents"
 )
 
-const jobIdString = "01f3j0g1md4qx7z5qb148qnh4r"
-const runIdString = "123e4567-e89b-12d3-a456-426614174000"
-
-var jobIdProto, _ = armadaevents.ProtoUuidFromUlidString(jobIdString)
-
-const jobSetName = "testJobset"
-const executorId = "testCluster"
-const nodeName = "testNode"
-const queue = "test-queue"
-const userId = "testUser"
-const priority = 3
-const updatePriority = 4
-const updateState = 5
-const podNumber = 6
-const jobJson = `{"foo": "bar"}`
-const jobProto = "hello world"
-const containerName = "testContainer"
-
-var baseTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:05.000Z")
-var updateTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:06.000Z")
-var startTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:07.000Z")
-var finishedTime, _ = time.Parse("2006-01-02T15:04:05.000Z", "2022-03-01T15:04:08.000Z")
+const (
+	jobIdString = "01f3j0g1md4qx7z5qb148qnh4r"
+	runIdString = "123e4567-e89b-12d3-a456-426614174000"
+)
+
```

Comment:
looks like it pulled jobIdproto (unused?) but wasn't smart enough to unify the two sets of const here